susemanager/src/mgr-salt-ssh

```diff
@@ -20,6 +20,8 @@ if __name__ == "__main__":
         os.chdir(tempfile.gettempdir())
         try:
             chugid("salt", "salt")
+            # Change HOME according to salt user
+            os.environ["HOME"] = os.path.expanduser("salt")
```

Comment:
@meaksh does this have to be `salt` or `~salt`?

Reply:
You are totally right! I wonder why it was working...

Reply:
it must be a path. In our case it should become /var/lib/salt

Reply:
```
If you're on Python 3.5+ you can use pathlib.Path.home():

from pathlib import Path
home = str(Path.home())
```